src/providers/boleto-api-caixa/index.js

```diff
@@ -118,6 +119,13 @@ const buildPayload = (boleto, operationId) => {
         stateCode: formattedRecipientStateCode,
       },
     },
+    payeeGuarantor: {
+      name: path(['company_name'], boleto),
+      document: {
+        type: getDocumentType(path(['company_document_number'], boleto)),
```

Comment:
opa, visto que o campo `company_document_number` não é obrigatório na super boleto, vale usarmos o pathOr

```suggestion
        type: getDocumentType(pathOr('',['company_document_number'], boleto)),
```

Reply:
feito